File: VB/WindowsApplication3/DateTapeEdit/DateHitInfo.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Drawing

Namespace WindowsApplication3
	Public Class DateHitInfo
		Public Sub New()
			MyBase.New()
		End Sub

		' Fields...
		Private _Point As Point
		Private _HitInfoType As DateInfoType

		Public Property HitInfoType() As DateInfoType
			Get
				Return _HitInfoType
			End Get
			Set(ByVal value As DateInfoType)
				_HitInfoType = value
			End Set
		End Property


		Public Property Point() As Point
			Get
				Return _Point
			End Get
			Set(ByVal value As Point)
				_Point = value
			End Set
		End Property

	End Class
End Namespace

File: CS/WindowsApplication3/DateTapeEdit/DateHitInfo.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Drawing;

namespace WindowsApplication3
{
    public class DateHitInfo
    {
        public DateHitInfo() : base() { }

        // Fields...
        private Point _Point;
        private DateInfoType _HitInfoType;

        public DateInfoType HitInfoType
        {
            get { return _HitInfoType; }
            set { _HitInfoType = value; }
        }


        public Point Point
        {
            get { return _Point; }
            set
            {
                _Point = value;
            }
        }
        
    }
}
